laravel/database/migrations/2020_06_25_113204_create_reserveertijden_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateReserveertijdenTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reserveertijden', function (Blueprint $table) {
+          $table->string('tijdsvak');
```

Comment:
Is het niet beter om te werken met begintijd / eindtijd? Volgens mij heeft @draZer0 dat ook zo gedaan met de openingstijden.

Reply:
ik zou een bigIncrement maken als tijdvak (een primary key / id)
een string begin tijd en een string eind tijd
en de double maxAantal veranderen naar integer max_aantal 

dan kan je in de seeder per tijdvak een begin- en eindtijd doen